### PR TITLE
ES-LA Translation

### PR DESCRIPTION
**Pull request by juanbeleno**

### CODE REVIEW

File: web-design-in-4-minutes.html [hunk 1]
```diff
@@ -214,8 +214,8 @@
             <em>Share on</em> <strong>Twitter</strong>
           </a>
         </nav>
-        <p>If you want to learn more about web design, check out <a href="http://marksheet.io" target="_blank"><strong>MarkSheet</strong>, my free HTML and CSS tutorial</a>.<br>
-        Or if you want to get started right away, try out <a href="http://bulma.io" target="_blank"><strong>Bulma</strong>, my CSS framework based on Flexbox</a>.</p>
+        <p>If you want to learn more about web design, check out <a href="http://marksheet.io" target="_blank"><strong>MarkSheet</strong></a>, my free HTML and CSS tutorial</a>.<br>
+        Or if you want to get started right away, try out <a href="http://bulma.io" target="_blank"><strong>Bulma</strong></a>, my CSS framework based on Flexbox</a>.</p>
```
Comment (jgthms) [inline web-design-in-4-minutes.html]: Why this change?

File: web-design-in-4-minutes-spanish.html [hunk 2]
```diff
@@ -0,0 +1,371 @@
+<!DOCTYPE html>
```
Comment (jgthms) [inline web-design-in-4-minutes-spanish.html]: Can you put this in a `/es` subfolder?

File: web-design-in-4-minutes.html (hunk 1 above)
Comment (jbeleno) [inline web-design-in-4-minutes.html]: Wow, it was my mistake, I thought it doesn't have </a> in the right place

File: web-design-in-4-minutes-spanish.html (hunk 2 above)
Comment (jbeleno) [inline web-design-in-4-minutes-spanish.html]: Done
